### ISSUE TITLE
[Identify map tool] add "Live update" option

### ISSUE THREAD

**Issue by janzandr** [EnMAP-Box/enmap-box#491]
_Requested by @dpoursanidis_.

Currently, the user has to **click a location inside the map view** to update the pixel profile and the cursor location values:
![image](https://github.com/EnMAP-Box/enmap-box/assets/45030715/dc14cc1d-d106-41e8-92f9-0b845a68efc6)

It is proposed to add a "Live update" option for the identify map tool, that **triggers the update while hovering over the map view**.

Depending on the datasources, a fluent live update may not be possible (due to data IO limitations), but for exploring a single image, it should work. At least I found a similar live update functionality in the SNAP Toolbox (at timestamp 3:10):
https://youtu.be/5znAQH6vrLs?t=190

Implementation should be very simple, just add a checkable toolbar button indicating live update on/off here:
![image](https://github.com/EnMAP-Box/enmap-box/assets/45030715/6803886f-3a5d-46f3-b850-b61214a8a931)
If live update is on, every **hover event** will be forwarded to the **left mouse clicked event**.

**Comment by janzandr** [comment]
@jakimowb, what do you think?

**Comment by jakimowb** [comment]
I expect performance bottlenecks if much data has to be loaded for each cursor position. 
In that case a timer could ensure that heavby-data loading tasks are triggered with longer intervals compared to updates of the GUI (e.g. the mouse cursor position)

**Comment by janzandr** [comment]
>  a timer could ensure that heavby-data loading tasks are triggered with longer intervals

Good idea. This should be a user-setting, so that I can crank it up for fast SSD drives, and slow it down for network drives.
